api/mod/src/settings.py

```diff
@@ -35,6 +36,7 @@ class Config:
 
 SETTINGS = Settings()
 if SETTINGS.database_url.startswith("postgres://"):
+    SETTINGS.engine_connect_args = {}
```

Comment:
I'm somewhat unsure whether that's the best way to do it... I see how it should always be `"check_same_thread": True` when using a postgres database. On the other hand, we could solve this without writing this additional line of code by specifying `engine_connect_args = {"check_same_thread": False}` in the dotenv file.

Besides that: This could overwrite some other part of the `engine_connect_args` when using the dotenv file, right? Then this solution seems a little radical to me - we should only overwrite the `"check_same_thread"` property and leave the rest as it is

Reply:
Oh I didnt know putting `engine_connect_args = {"check_same_thread": False}` in the dotenv is possible, gonna do that then ^^

Reply:
Im not sure but I think postgres already threw an error not because the value is True or False, but because the setting is not valid, so the dict must not contain the key I think

Reply:
Okidoki
We could also simply add the `engine_connect_args` option to our `.env.dev` file and leave it unset when in staging?

Reply:
So what I mean: Instead of wiping the engine_connect_args in staging + production, only set them in `.env.dev`

Reply:
Think so too 👍